src/map.react.js

```diff
@@ -540,17 +534,13 @@ var MapGL = React.createClass({
     var pos = opt.pos;
 
     // Radius enables point features, like marker symbols, to be clicked.
-    map.featuresAt([pos.x, pos.y], {
-      radius: 15
-    }, function callback(error, features) {
-      if (error) {
-        throw error;
-      }
-      if (!features.length) {
-        return;
-      }
-      this.props.onClickFeatures(features);
-    }.bind(this));
+    var size = 5;
```

Comment:
5 is going to be a little bit less than 15 (box extent of 10). Was 15 too large a value?